src/test/stellar/test-auth-register.ts

```
@@ -12,58 +12,229 @@ interface RegisterResponse {
+      data.user &&
+      typeof data.user.email === 'string' &&
+      typeof data.user.authMethod === 'string' &&
+      typeof data.user.publicKey === 'string'
+    );
+  }
+
+  private async cleanup(): Promise<void> {
+    try {
+    } catch (error) {
+      console.warn(`Cleanup warning: ${error instanceof Error ? error.message : String(error)}`);
+    }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)